js/app/modules/card/thumb.js

```diff
@@ -113,6 +113,9 @@ const Thumb = new Module.Class({
     _init: function (props={}) {
         this.parent(props);
 
+        if (this.model.content_type === 'application/pdf')
```

Comment:
Might be nicer to do this generally, some way of adding style classes based on content type for all cards not just thumb.